i18n/locales/ar.json

```diff
@@ -34,6 +34,8 @@
   "components.header.buttons.toggleDarkMode": "Toggle dark/light mode",
   "components.pagination.next": "التالي | ",
   "components.pagination.previous": "السابق",
+  "components.author.githubLinkLabel": "{username} Github - opens in new tab",
```

Comment:
This should also be added in the bottom. Any new translation key should be added in the bottom.

Reply:
Also note, please only keep `en.json` as we discussed earlier. We don't want to add the new translation keys on the other language files. Crowdin will do that eventually for us :)